lib/customerio/util.ex

```diff
@@ -20,6 +22,14 @@ defmodule Customerio.Util do
   """
   @type method :: :get | :post | :delete | :put | :patch
 
+  def send_behavioral_request(method, route, data_map, opts \\ []) do
```

Comment:
Let's add some specs to these methods.

```suggestion
@spec ...
  def send_behavioral_request(method, route, data_map, opts \\ []) do
```

Reply:
👍